src/BlazorUI/Bit.BlazorUI/Components/Buttons/ToggleButton/BitToggleButton.scss

```
@@ -15,25 +15,128 @@
+.bit-tgb-end {
+    flex-direction: row;
+}
+
+.bit-tgb-spn {
+    border-radius: 50%;
+    border-width: spacing(0.2125);
+    width: var(--bit-tgb-spn-size);
+    height: var(--bit-tgb-spn-size);
+    border-color: $clr-brd-sec;
+    border-style: $shp-border-style;
+    // the button's own foreground already contrasts with its background in every variant and state
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)